paseos/visualization_example/example_animation.py

```diff
@@ -0,0 +1,16 @@
+import sys
+
+sys.path.append("../..")
+
+from utils import test_setup
+from paseos.visualization.space_animation import SpaceAnimation
+
+
+def animation():
```

Comment:
This is not a very instructive example as everything important (creating the actors) is abstracted away and there is no documentation describing what we are actually doing. Would at least add comments on

* short description what this example shows
* what the important steps are

Reply:
(if you want to show the same example then let's at least a comment to see there for scenario explanation)

Reply:
I removed the util file and added a short summary.